### ISSUE TITLE
infer_table_from_schema! panics when that table is in a non-default schema

### ISSUE THREAD

**Issue by jasonl** [diesel-rs/diesel#646]
Using version 0.10.0:

```
error: proc-macro derive panicked
 --> src/models/mrr_interval.rs:8:1
  |
8 | infer_table_from_schema!("dotenv:PG_URL", "snowflake.mrr_intervals");
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Postgres TableDate has schema
  = note: this error originates in a macro outside of the current crate
```

The panic comes from:

https://github.com/diesel-rs/diesel/blob/v0.10.0/diesel_infer_schema/src/pg.rs#L131

Which is because (as far as I can see), `TableData.schema` can only ever be `None` as set at:

https://github.com/diesel-rs/diesel/blob/v0.10.0/diesel_codegen/src/schema_inference.rs#L58

A brief look at `master` seems to suggest this might already be fixed, but I'll raise this issue anyway so that the developers are aware of it.

This is being built on a recent nightly, but I'm not sure that's relevant.

```
ubuntu@snowflake:/vagrant$ rustc --version
rustc 1.17.0-nightly (c49d10207 2017-02-07)
```

**Comment by sgrif** [comment]
Yes, this has been fixed on master. It's a serious enough bug to warrant a new release. I will cherry-pick it and release a bugfix release with it.

**Comment by sgrif** [comment]
I've released v0.10.1 which includes a fix for this issue. For posterity, this was fixed by https://github.com/diesel-rs/diesel/commit/e0272a4398444666ea7a62b2cf1e6a98003fd17e

**Comment by jasonl** [comment]
Thanks for the quick response!
